src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -10,12 +10,27 @@ import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
  *     <li>shapes
  */
 export default class MxGraphConfigurator {
+  private mxClient: any = MxGraphFactoryService.getMxGraphProperty('mxClient');
+  private mxGraph: any = MxGraphFactoryService.getMxGraphProperty('mxGraph');
+  private mxGraphModel: any = MxGraphFactoryService.getMxGraphProperty('mxGraphModel');
   private mxUtils: any = MxGraphFactoryService.getMxGraphProperty('mxUtils');
   private mxConstants: any = MxGraphFactoryService.getMxGraphProperty('mxConstants');
   private mxPerimeter: any = MxGraphFactoryService.getMxGraphProperty('mxPerimeter');
-  constructor(readonly graph: mxgraph.mxGraph) {}
+  private mxShape: any = MxGraphFactoryService.getMxGraphProperty('mxShape');
 
-  public configureStyles(): void {
+  private readonly graph: mxgraph.mxGraph;
+
+  constructor(container: Element) {
+    this.initMxGraphPrototype();
+    this.graph = new this.mxGraph(container, new this.mxGraphModel());
+    this.configureStyles();
+  }
+
+  public getGraph(): mxgraph.mxGraph {
+    return this.graph;
+  }
+
+  private configureStyles(): void {
```

Comment:
To not have a bigger class, I think it would be better to move the style configuration in an other class, like StyleConfigurator :)

Reply:
sure